src/TunNetCom.SilkRoadErp.Sales.Api/Features/Products/DeleteProduct/DeleteProductEndpoint.cs

```diff
@@ -0,0 +1,21 @@
+﻿namespace TunNetCom.SilkRoadErp.Sales.Api.Features.Products.DeleteProduct
+{
+    public class DeleteProductEndpoint : ICarterModule
+    {
+        public void AddRoutes(IEndpointRouteBuilder app)
+        {
+            app.MapDelete("/Products/{refe}", async Task<Results<NoContent, NotFound>> (IMediator mediator, string refe) =>
```

Comment:
{refe:string}

Add CancellationToken

Reply:
{refe:string} : make error  
so i asked chatgpt and tell me that we don't need to explicitly specify type